doc/subsystems/settings/settings.rst

```diff
@@ -58,6 +57,17 @@ initializes the FCB area, so it must be called before calling
 ``settings_fcb_dst()``. File read target is registered using
 ``settings_file_src()``, and write target by using ``settings_file_dst()``.
 
+Loading data from persisted storage
+***********************************
+
+Call to ``settings_load()`` causes loading data from the storage to its volatile
```

Comment:
The travelling through the persistent storage is dictated by the backend. If a backend implements the settings_load in a different way, repeated calls to h_set() could be avoided.
The h_commit() allows the settings routines to first retrieve settings from storage and then to apply them all at once.

Reply:
I made a note that it is for back-end we already have.